types/para_ids.go

```diff
@@ -0,0 +1,5 @@
+package types
```

Comment:
`para_ids` seems like an odd file name to me. Is `para` meant to be `param`?

Reply:
It is weird, and outdated. Renamed.